src/gui/uithememanager.cpp

```diff
@@ -168,9 +168,6 @@ void UIThemeManager::initInstance()
 
 UIThemeManager::UIThemeManager()
     : m_useCustomTheme(Preferences::instance()->useCustomUITheme())
-#if (defined(Q_OS_UNIX) && !defined(Q_OS_MACOS))
-    , m_useSystemTheme(Preferences::instance()->useSystemIconTheme())
```

Comment:
You also need to remove all other usages of `m_useSystemTheme` from this file and its declaration from `uithememanager.h`.